document API endpoints for osquery and orbit (#8536)
fleetdm/fleet: docs/Contributing/FAQ.md

Failed to run docker-compose
```

> You are probably running an old version of Docker. You should download the installer for your platform from https://docs.docker.com/compose/install/

Reviewer: ```suggestion
You're probably running an old version of Docker. Download the installer for your platform from the [Docker Documentation](https://docs.docker.com/compose/install/).
```

> ## What API endpoints does osquery and orbit need access to?

Reviewer: ```suggestion
## What API endpoints do osquery and Orbit need access to?
```

Based on the configuration, osquery running on hosts will need access to these API endpoints:

* `/api/v1/osquery/enroll`
* `/api/v1/osquery/config`
* `/api/v1/osquery/distributed/read`
* `/api/v1/osquery/distributed/write`
* `/api/v1/osquery/carve/begin`
* `/api/v1/osquery/carve/block`
* `/api/v1/osquery/log`

> In addition to above, if you have orbit running on hosts it will need access to these API endpoints:

Reviewer: ```suggestion
If you also have Orbit running on hosts, it will need access to these API endpoints:
```
